tests/devapps/WAM/NetCoreWinFormsWam/Form1.cs

```diff
@@ -295,46 +317,48 @@ private bool IsMsaPassthroughConfigured()
             return msa;
         }
 
-        private Prompt GetPrompt()
+        private Prompt? GetPrompt()
```

Comment:
This is interesting. What does it imply? Return value is optional?

Reply:
Nullable values. In C#, as in Java, value types (int, bool, enums, structs like Prompt) cannot be null. C# added the concept of "nullable value", where you add a `?` after the type, e.g. `int?` means nullable int etc. You can then do things like: 

```csharp
int? i = null; 
int? j = 14;

i.HasValue // false
j.HasValue // true
j.Value // 14
i.Value // exception I think

if (i == 14) { .. } // is equivalent to 
if (i.HasValue && i.Value ==14) { ...} 

For more details see https://docs.microsoft.com/en-us/dotnet/api/system.nullable-1.value?view=net-5.0

```